structure-message/requirements/browse-by-topic.md

```diff
@@ -0,0 +1,83 @@
+# Requirements: Browse by topic
```

Comment:
Images would not show. I could not see them, even when navigating to the corresponding folder: 
https://github.com/sdmx-twg/sdmx-json/blob/structure-message-release/structure-message/requirements/img